src/TabSwipe/widget/TabSwipe.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<widget id="TabSwipe.widget.TabSwipe" needsEntityContext="false" xmlns="http://www.mendix.com/widget/1.0/" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.mendix.com/widget/1.0/ ../../../xsd/widget.xsd">
+    <name>Tab Swipe</name>
+    <description>Swipe to navigate between tabs</description>
+    <icon/>
+    <properties>
+        <property key="targetName" type="string">
+            <caption>Target tab name</caption>
+            <category>General</category>
+            <description>Name of the tab-container. Place this widget directly below the tab container</description>
```

Comment:
Does the location makes a different? `Place this widget directly below the tab container`